doesn't work: custom repo when creating switch

In the [manual](https://opam.ocaml.org/doc/man/opam-switch.html):

> Prepare a local switch for building the packages defined in ./. This scans the current directory for package definitions, chooses a compatible compiler, creates a local switch and installs the local package dependencies.
> ```
> opam switch create trunk --repos default,beta=https://github.com/ocaml/ocaml-beta-repository.git ocaml-variants.4.10.0+trunk
> ```

This command fails with:

```
Creating repository beta...
[ERROR] Could not update repository "beta": OpamDownload.Download_fail(_, "curl: code 404 while
        downloading https://github.com/ocaml/ocaml-beta-repository.git/index.tar.gz")
[ERROR] Initial fetch of these repositories failed: beta
```

I also tried with my own custom repo and it fails with the same error.

(opam version 2.0.8)

--- a/master_changes.md
+++ b/master_changes.md
@@ -37,6 +37,7 @@ New option/command/subcommand are prefixed with ◈.
   * Don't exclude base packages from rebuilds (made some sense in opam 2.0
     with base packages but doesn't make sense with 2.1 switch invariants) [#4569 @dra27]
   * Don't refer to base packages in messages any more [#4623 @dra27 - fixes #4572]
+  * Give the correct command when demonstrating switch creation [#4675 @dra27 - fixes #4673]
 
 ## Pin
   * Don't look for lock files for pin depends [#4511 @rjbou - fix #4505]

--- a/src/client/opamCommands.ml
+++ b/src/client/opamCommands.ml
@@ -2436,7 +2436,7 @@ let switch cli =
           compatible compiler, creates a local switch and installs the local \
           package dependencies.";
       `Pre "    opam switch create trunk --repos \
-            default,beta=https://github.com/ocaml/ocaml-beta-repository.git \
+            default,beta=git+https://github.com/ocaml/ocaml-beta-repository.git \
             ocaml-variants.4.10.0+trunk";
       `P "Create a new switch called \"trunk\", with \
           $(b,ocaml-variants.4.10.0+trunk) as compiler, with a new $(i,beta) \

